app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Since you have the reference in the migration you should include the foreign_keys.

Reply:
Ok thank you will do that